### PR TITLE
kick-start work

### PR DESCRIPTION
**Pull request by HanbaiWang**
I have completed the schedule and meeting links of CTD Sub-project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Repository to describe, develop, document and test the ClickToDial API family
 
 ## Meetings
 * Meetings are held virtually
-* Schedule: tbd
-* Meeting link: tbd
+* Schedule: bi-weekly (odd weeks), Wednesday, 1 PM UTC+8
```
Comment (hdamker) [inline README.md:21]: If based on UTC, then maybe 05:00 UTC is more clear (local time zone could be mentioned in addition). 
BTW: will it change with daylight saving time, or is fixed to UTC? So will it be 07:00 CEST and later 06:00 CET?

**Review by hdamker — APPROVED** [review]
LGTM, see comment on time zone notation

File: README.md [hunk 2]
```diff
@@ -18,8 +18,8 @@ Repository to describe, develop, document and test the ClickToDial API family
 
 ## Meetings
 * Meetings are held virtually
-* Schedule: tbd
-* Meeting link: tbd
+* Schedule: bi-weekly (odd weeks), Wednesday, 3 PM CET/CEST (13:00 UTC,21:00 Beijing)
```
Comment (hdamker) [inline README.md:21]: Thanks for the clarification.

**Review by hdamker — APPROVED** [review]

**Comment by ShutingQing** [comment]
LGTM. @DanXu-ChinaTelecom Dan, pls take a look.

**Comment by ShutingQing** [comment]
@HanbaiWang hanbai, can you add me or dan as reviewer, i believe when 2 reviewers reivew, then this can be merged.

**Review by ShutingQing — APPROVED** [review]

**Comment by DanXu-ChinaTelecom** [comment]
LGTM.

**Comment by hdamker** [comment]
@HanbaiWang the PR is ready to be merged, I suppose.

**Pull request merged**